docs/source/reservations/reservations.rst

```diff
@@ -0,0 +1,182 @@
+
+.. hint::
+
+    It has been observed that using DWS can significantly increase the chance of getting a high-end GPU resource, such as A100s and H100s, compared to using on-demand or spot instances.
+
+
+Using DWS for VMs
+~~~~~~~~~~~~~~~~~
+
+SkyPilot allows you to launch a cluster with DWS by specifying the ``gcp.managed_instance_group`` field in ``~/.sky/config.yaml``:
+
+.. code-block:: yaml
+
+    gcp:
+      managed_instance_group:
+        run_duration: 3600
+        provision_timeout: 900
+
+The ``run_duration`` is the duration for a created instance to be kept alive (in seconds, required), and the ``provision_timeout`` is the timeout for provisioning an instance with DWS (in seconds, optional). If the timeout is reached without getting the requested resources, SkyPilot will automatically :ref:`failover <auto-failover>` to other clouds/regions/zones to get the resources.
+
+See the :ref:`config-yaml` for more details.
+
+In case you want to specify the DWS configuration for each job/cluster, you can also specify the configuration in the SkyPilot task YAML (see :ref:`here <task-yaml-experimental>`):
+
+.. code-block:: yaml
+    experimental:
+      config_overrides:
+        gcp:
+          managed_instance_group:
+            run_duration: 3600
+            provision_timeout: 900
+
+    resources:
+      cloud: gcp
+      accelerators: A100:8
+    
+    num_nodes: 4
+    
+Using DWS on GKE with Kueue
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+DWS is also supported on Google Kubernetes Engine (GKE) with Kueue. To enable DWS on GKE, you need to set up your GKE cluster with Kueue and DWS, see the `GCP official docs <https://cloud.google.com/kubernetes-engine/docs/how-to/provisioningrequest>`
+
+To launch a SkyPilot cluster or job on GKE with DWS, you can specify the DWS configuration in the SkyPilot task YAML:
+
+.. code-block:: yaml
+
+    experimental:
+      config_overrides:
+        kubernetes:
+          pod_config:
+            metadata:
+              annotations:
+                provreq.kueue.x-k8s.io/maxRunDurationSeconds: "3600"
+          provision_timeout: 900
+
+    resourcse:
+      cloud: kubernetes
+      accelerators: A100:8
+      labels:
+        kueue.x-k8s.io/queue-name: dws-local-queue
+
+Here, ``kueue.x-k8s.io/queue-name`` is the name of the Kueue queue to submit your resource request to, and ``provreq.kueue.x-k8s.io/maxRunDurationSeconds`` is the maximum duration for a created instance to be kept alive (in seconds, required), and ``provision_timeout`` is the timeout for provisioning an instance with DWS (in seconds, optional). If the timeout is reached without getting the requested resources, SkyPilot will automatically :ref:`failover <auto-failover>` to other clouds/regions/zones to get the resources.
```

Comment:
split these 3 fields into 3 bullets for readability?

for last sentence, see comment above?